mchf-eclipse/drivers/audio/audio_nr.h

```diff
@@ -39,7 +39,7 @@ typedef struct __MCHF_SPECIALMEM NoiseReduction
 	//											 3 = display bin gain multiplied with long_tone_gain
 } NoiseReduction;
 
-NoiseReduction NR;
+NoiseReduction __MCHF_SPECIALMEM NR;
 
```

Comment:
This is the culprit. I did not notice yesterday: You should not define variables in header files, only declare. See
https://stackoverflow.com/questions/1410563/what-is-the-difference-between-a-definition-and-a-declaration
In audio_nr.h you declare:
```
extern +NoiseReduction NR;
```
In audio_nr.c (or any other convenient place) you define:
```
NoiseReduction __MCHF_SPECIALMEM NR;
```
BTW it gets even worse if you define (!) something as "static" (e.g. "static int am_i_here_multiple_times") in a header file. In each compilation unit (aka c file) you would get a different "am_i_here_multiple_times" variable AND no compiler warning. 
The C language isn't an easy language, is it?